cantools/database/can/message.py

```diff
@@ -220,13 +232,28 @@ def signal_groups(self, value):
     def comment(self):
         """The message comment, or ``None`` if unavailable.
 
+        Note that we implicitly assume the comment's language to be english.
+        
         """
+        if self._comments is None:
+            return None
+        return self._comments.get("EN")
+
+    @property
+    def comments(self):
+        """The dictionary with the descriptions of the message in multiple languages. ``None`` if unavailable.
 
-        return self._comment
+        """
+        return self._comments
 
     @comment.setter
     def comment(self, value):
-        self._comment = value
+        # we implicitly assume english as the language of "the" comment
+        self._comments = { "EN": value }
```

Comment:
Must handle setting the comment to `None`.

Reply:
Done in the latest revision. (I hope)